mock/operations/material-aid.ts

```diff
@@ -13,3 +13,8 @@ export function materialAidGet(req: express.Request, res: express.Response): exp
   const { id } = req.params;
   return res.json(materialAidList.content?.find((el) => el.id === +id));
 }
+
+export function materialAidDelete(req: express.Request, res: express.Response): express.Response {
+  const { id } = req.params;
+  return res.json(materialAidList.content?.find((el) => el.id === +id));
```

Comment:
This is the same as `Get`; that doesn't look correct. Can you check what's the real API response for a DELETE request and adjust the implementation here to be similar?

The same for other `Delete` places.